src/static/js/chapter.js

```diff
@@ -1,3 +1,61 @@
+//This function removes the lazy-loading attributes from all img and iframe tags
+//Useful for print view for example (https://bugs.chromium.org/p/chromium/issues/detail?id=875403)
+function removeLazyLoading() {
+
+  console.log("Removing lazy loading...");
+
+  var all_imgs = document.querySelectorAll('img');
+  for (index = 0; index < all_imgs.length; ++index) {
+    var img = all_imgs[index];
+    if (img.getAttribute('loading')) {
+      img.removeAttribute('loading');
+    }
+  }
+  
+  var all_iframes = document.querySelectorAll('iframe');
+  for (index = 0; index < all_iframes.length; ++index) {
+    var iframe = all_iframes[index];
+    if (iframe.getAttribute('loading')) {
+      iframe.removeAttribute('loading');
+    }
+  }
+}
+
+//Add an event handler to remove LazyLoading when entering print mode
+function removeLazyLoadingOnPrint() {
+  if ("onbeforeprint" in window) {
+    window.onbeforeprint = function() {

```

Comment:
Could be simplified
```suggestion
    window.onbeforeprint = removeLazyLoading;
```

Reply:
Did try this but couldn’t get it to work. Will give it another go in case I was doing something stupid.

Reply:
You could be right. It should work in theory but IIRC you need to give it context, eg `removeLazyLoading.bind(window)` or something silly.

Reply:
No this works fine. Must have been doing something stupid.